src/hooks/constants/queries/useGetConstantsQuery/useGetConstantsQuery.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { ApiPromise } from '@polkadot/api';
```

Comment:
automocking

Reply:
locate next to resolver composition

Reply:
set up composition to be reused with .each ( I think there are examples of this in codebase now.. )